tests/module/environmental/src/main.c

```diff
@@ -129,22 +113,6 @@ void setUp(void)
 	zbus_sub_wait(&battery, &chan, K_NO_WAIT);
 }
 
-void test_all_zeroes(void)
```

Comment:
Unrelated change. Also not mentioned in the commit.

Reply:
not unrelated - this test is not compatible with the change

Reply:
added this to the commit message - cannot make this a fail-expected test because the module is forcing a crash in that case.